.github/workflows/test.yml

```
@@ -15,7 +15,7 @@ jobs:
             - name: Use Node.js
               uses: actions/setup-node@v1
               with:
-                  node-version: 14.x
+                  node-version: 16.x
```

And with the updated action you can source the Node version from `.nvmrc`:
```suggestion
                  node-version-file: '.nvmrc'
```